msktutil.cpp

```diff
@@ -495,6 +500,9 @@ void do_help()
     fprintf(stdout, "                         given service account. The service is of the form\n");
     fprintf(stdout, "                         <service>/<hostname>.\n");
     fprintf(stdout, "                         If the hostname is omitted, assumes current hostname.\n");
+    fprintf(stdout, "                         Default for machine accounts:\n");
+    fprintf(stdout, "                         host/long_hostname and host/short_hostname.\n");
+    fprintf(stdout, "                         Default for service accounts: None");
```

Comment:
A similar note should be added to the man page as well.

Reply:
:+1: